{"name": "test_pytest_rerunfailures[2-failed]", "status": "passed", "attachments": [{"name": "stdout", "source": "4ee8a271-aaa7-4ae3-9365-bc5aff0337ac-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "countdown", "value": "2"}, {"name": "status", "value": "'failed'"}], "start": 1710872971224, "stop": 1710872971250, "uuid": "82902d3a-24ca-429e-8b4d-b4d3a8871623", "historyId": "2c978fa4b8fef3616deaea099a64c070", "testCaseId": "df06b7d101485b30103206befbeb4d8a", "fullName": "tests.allure_pytest.externals.pytest_rerunfailures.pytest_rerunfailures_test#test_pytest_rerunfailures", "labels": [{"name": "feature", "value": "Integration"}, {"name": "parentSuite", "value": "tests.allure_pytest.externals.pytest_rerunfailures"}, {"name": "suite", "value": "pytest_rerunfailures_test"}, {"name": "host", "value": "MacBook-Pro-whenessel.local"}, {"name": "thread", "value": "29518-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.allure_pytest.externals.pytest_rerunfailures.pytest_rerunfailures_test"}], "links": [{"type": "issue", "url": "140", "name": "140"}]}